oversweet-domain/src/main/java/com/depromeet/oversweet/domain/bookmark/entity/DrinkBookmarkEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.depromeet.oversweet.domain.bookmark.entity;
+
+import com.depromeet.oversweet.domain.common.entity.BaseTimeEntity;
+import com.depromeet.oversweet.domain.drink.entity.DrinkEntity;
+import com.depromeet.oversweet.domain.member.entity.MemberEntity;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Table(name = "drink_bookmark")
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class DrinkBookmarkEntity extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
```

Comment:
외래키 연관 관계 같은 경우는 JPA에서 랜덤한 문자열로 제약 조건 이름을 갖게 되는데 저희가 직접 제어하면 어떨까요??

```suggestion
    @JoinColumn(name = "member_id", nullable = false,  foreignKey = @ForeignKey(name = "fk_drinkBookMark_to_member"))
```

Reply:
이 부분은 제가 다시 살펴보고 수정해두겠습니다. 감사합니다!